index.html

```diff
@@ -28,5 +29,93 @@ <h1>I’m Amine Glade to see you!</h1>
       </ul>
   </section>
+  <!-- Begin The ToolBar and the Headline-->
+
+  <!-- Begin Work section-->
+  <section class="Work">
+    <div class="item Work-1">
+      <img src="/Work-Profolio/1.png" alt="Tonic">
+      <h1>Tonic</h1>
+      <ul class="List">
+        <li class="List-1st">CANOPY</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="Work-List">
+        <li>html</li>
+        <li>css</li>
+        <li>javaScript</li>
+      </ul>
+      <h3>See Project</h3>
+    </div>
+    <div class="item Work-2">
+      <img src="/Work-Profolio/2.png" alt="Multi-Post Stories">
+      <h1>Multi-Post Stories</h1>
+      <ul class="List">
+        <li class="List-1st">CANOPY</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="Work-List">
+        <li>html</li>
+        <li>css</li>
+        <li>javaScript</li>
+      </ul>
+      <h3>See Project</h3>
+    </div>
+    <div class="item Work-2">
+      <img src="/Work-Profolio/3.png" alt="Tonic-Canopy">
+      <h1>Tonic</h1>
+      <ul class="List">
+        <li class="List-1st">CANOPY</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="Work-List">
+        <li>html</li>
+        <li>css</li>
+        <li>javaScript</li>
+      </ul>
+      <h3>See Project</h3>
+    </div>
+    <div class="item Work-3">
+      <img src="/Work-Profolio/4.png" alt="Multi-Post Stories-Canopy">
+      <h1>Multi-Post Stories</h1>
+      <ul class="List">
+        <li class="List-1st">CANOPY</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="Work-List">
+        <li>html</li>
+        <li>css</li>
+        <li>javaScript</li>
+      </ul>
+      <h3>See Project</h3>
+    </div>
+  </section>
+  <!-- End Work section-->
+  <!-- Begin About me Section-->
+  <section class="headline">
+    <h1>About <br>Myself  </h1>
+      <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+      </p>
+      <span><a>LET'S CONNECT <br> </a></span>
+      <ul class="Social">
+        <li><a href="#"><img src="/Icons/twitter.png" alt="twitter"></a></li>
+        <li><a href="#"><img src="/Icons/github.png" alt="GitHub"></a></li>
+        <li><a href="#"><img src="/Icons/angellist.png" alt="angellist"></a></li>
+        <li><a href="#"><img src="/Icons/medium.png" alt="medium"></a></li>
+      </ul>
+      <h3>Get my resume</h3>
```

Comment:
- [ ] You haven't implemented all the parts of the `About` section. Please add the languages, frameworks, and skills to the `About` section.